src/app/component/ExpandableContent/ExpandableContent.component.js

```diff
@@ -66,11 +68,45 @@ export class ExpandableContent extends PureComponent {
         return null;
     }
 
+    scrollToExpandedContent(elementToScrollRef, isContentExpanded, coveringElementsIds) {
+        const elem = elementToScrollRef && elementToScrollRef.current;
+
+        requestAnimationFrame(() => {
+            if (!isContentExpanded && elem) {
+                const elemToWindowTopDist = elem.getBoundingClientRect().top;
+                const windowToPageTopDist = document.body.getBoundingClientRect().top;
+                const topToElemDistance = elemToWindowTopDist - windowToPageTopDist;
+
+                const coveringElementsHeight = coveringElementsIds.reduce((acc, elementId) => {
+                    // eslint-disable-next-line no-param-reassign
+                    acc += document.getElementById(elementId).offsetHeight;
+
+                    return acc;
+                }, 0);
+
+                const scrollTo = topToElemDistance - (screen.height - coveringElementsHeight - elem.offsetHeight);
+
+                // checking if button is in a view-port
+                if (-windowToPageTopDist < scrollTo) {
+                    // window.scrollTo(0, scrollTo);
+                    window.scrollTo(0, scrollTo);
+                }
+            }
+        });
+    }
+
     toggleExpand = () => {
         const { onClick } = this.props;
+        const { isContentExpanded } = this.state;
+
         if (onClick) {
             onClick(); return;
         }
+
+        this.scrollToExpandedContent(
```

Comment:
I believe these arguments are not necessary here, due to all of them being retrieved from `this`